README.md

```diff
@@ -1,10 +1,33 @@
 # Minimum GHC Installer [![Build Status](https://img.shields.io/travis/fpco/minghc.svg?style=flat)](https://travis-ci.org/fpco/minghc)
 
+## Wait, are you sure you don't want `stack`?
+
+For most use cases we actually recommend [you download and use
+`stack`](https://github.com/commercialhaskell/stack/wiki/Downloads#windows)
+instead of proceeding with MinGHC. The MinGHC approach to installing both GHC
+and MSYS has been adopted by `stack` and `stack` provides some additional
+benefits:
+
+* Easier to upgrade to newer GHC versions
+* Support for working with multiple GHC versions without switching environments
+* Simple upgrade procedure for `stack` itself
+* Isolated environment that avoids contaminating your `PATH` variable
+
+For more information on using `stack`, please [read the `stack`
+guide](https://github.com/commercialhaskell/stack/blob/master/GUIDE.md).
+
+## MinGHC
+
+MinGHC is still a supported and active project, and makes sense for people
+looking for network-free installers for a Haskell toolchain. For more
```

Comment:
Why not make an explicit exception in this list saying "if you need an older version of GHC, use MinGHC."